src/app-logic/constants.js

```diff
@@ -97,6 +97,8 @@ export const PROFILER_SERVER_ORIGIN = 'https://api.profiler.firefox.com';
 // This is your local server:
 // export const PROFILER_SERVER_ORIGIN = 'http://localhost:5252';
 
+export const SYMBOL_SERVER_URL = 'https://symbols.mozilla.org';
```

Comment:
nit: please add a comment, eg:
```
// This is the default server.
// You can change it with the URL parameter `symbolServer=XXX`.
// See XXX for instructions about running a local server, or use
// `symbolication.stage.mozaws.net` for the official staging server. 
```

(I'm not sure all this comments shoul dbe here, or if some of it should be around `_shouldAllowSymbolServerUrl`)